binutils/binutils-2.26.def

```diff
@@ -0,0 +1,62 @@
+# Author: Paulo Souza, Shahzeb Siddiqui
+
+# docker-bootstrap:
+# Copyright (c) 2015-2016, Gregory M. Kurtzer. All rights reserved.
+# 
+# "Singularity" Copyright (c) 2016, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory (subject to receipt of any
+# required approvals from the U.S. Dept. of Energy).  All rights reserved.
+
+BootStrap: docker
+From: centos
+Include: yum
+
+
+# eb-gcc:
+%post
+		export EASYBUILD_INSTALLPATH=/app
+
+		# scratch space to build and download source files from eb
+		export EASYBUILD_PREFIX=/scratch
+
+		mkdir -p $EASYBUILD_PREFIX
+                mkdir -p $EASYBUILD_INSTALLPATH
+
+                # install csh/tcsh shell
+                # yum install tcsh -y
+ 
+                # epel.repo needed for Lmod and python2-pip
+                yum install -y epel-release
+                yum install -y Lmod
+
+                # eb dependencies
+                yum install python-devel -y
+                yum install python-setuptools -y
```

Comment:
being installed with `pip` below, so either drop this, or drop the `pip install setuptools`?